go/vt/vttablet/tabletserver/stream_consolidator_test.go

```diff
@@ -330,14 +329,14 @@ func TestConsolidatorMemoryLimits(t *testing.T) {
 
```

Comment:
Does `streamItemDelay` need to be 10ms above or could that be much lower at 1ms to also have a faster text execution? And the sleep window below could be kept the same, or even lower with more of a time buffer?

Reply:
```patch
diff --git i/go/vt/vttablet/tabletserver/stream_consolidator_test.go w/go/vt/vttablet/tabletserver/stream_consolidator_test.go
index 97c68aeac3..f3516999a1 100644
--- i/go/vt/vttablet/tabletserver/stream_consolidator_test.go
+++ w/go/vt/vttablet/tabletserver/stream_consolidator_test.go
@@ -323,13 +323,13 @@ func TestConsolidatorMemoryLimits(t *testing.T) {
        t.Run("two-phase consolidation (time)", func(t *testing.T) {
                ct := consolidationTest{
                        cc:              NewStreamConsolidator(128*1024, 2*1024, nocleanup),
-                       streamItemDelay: 10 * time.Millisecond,
+                       streamItemDelay: 1 * time.Millisecond,
                        streamItemCount: 10,
                }
 
                ct.run(10, func(worker int) (string, StreamCallback) {
                        if worker > 4 {
-                               time.Sleep(150 * time.Millisecond)
+                               time.Sleep(50 * time.Millisecond)
                        }
                        return "select 1", func(_ *sqltypes.Result) error {
                                return nil
```